NIMATRO/NIMATRO-IOC-01App/Db/controls.db

```diff
@@ -0,0 +1,422 @@
+}
+
+
+# Initially generated by $Id$
+# 
+# auto-generated EPICS records specify an asyn port "lvfp", but this can be changed - it just needs
+# to match the first argument of the relevant lvDCOMConfigure() command in the IOC st.cmd 
+# 
+      
+# 
+# Definitions from configSection "frontpanel" in XML input file
+#
+#     lvDCOMConfigure("lvfp", "frontpanel", "/path/to/lvinput.xml")
+# 
+
+# Write to LabVIEW control "Speed" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)SPEED")
+{
+	field(DESC, "LabVIEW 'Speed'")
+    field(DTYP, "asynFloat64")
+	field(SCAN, ".5 second")
+    field(INP,  "@asyn(lvfp,0,0)Speed")
+    field(PREC, "3")
+    field(EGU, "cm^2/min")
+    info(INTEREST, "HIGH")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:SPEED")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)SPEED:SP")
+{
+	field(DESC, "LabVIEW 'Speed'")
+    field(DTYP, "asynFloat64")
+    field(OUT,  "@asyn(lvfp,0,0)Speed")
+    field(PREC, "3")
+    field(EGU, "cm^2/min")
+	field(DRVH, "174.2")
+	field(DRVL, "-174.2")
+    info(INTEREST, "HIGH")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:SPEED:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+alias("$(P)SPEED", "$(P)SPEED:SP:RBV")
+
+# Write to LabVIEW control "Target Pressure" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)PRESSURE:TARGET")
+{
+	field(DESC, "LabVIEW 'Target Pressure'")
+    field(DTYP, "asynFloat64")
+    field(INP,  "@asyn(lvfp,0,0)Target_Pressure")
+    field(PREC, "3")
+    field(SCAN, ".5 second")
+    field(EGU, "mN/m")
+    info(INTEREST, "MEDIUM")
+    info(archive, "VAL")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PRESSURE:TARGET")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)PRESSURE:TARGET:SP")
+{
+	field(DESC, "LabVIEW 'Target Pressure'")
+    field(DTYP, "asynFloat64")
+    field(OUT,  "@asyn(lvfp,0,0)Target_Pressure")
+    field(PREC, "3")
+    field(EGU, "mN/m")
+	field(DRVH, "75.0")
+	field(DRVL, "-75.0")
+    info(INTEREST, "MEDIUM")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PRESSURE:TARGET:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+alias("$(P)PRESSURE:TARGET", "$(P)PRESSURE:TARGET:SP:RBV")
+
+# Read LabVIEW control/indicator "pressure" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)PRESSURE")
+{
+	field(DESC, "LabVIEW 'pressure'")
+    field(DTYP, "asynFloat64")
+    field(INP,  "@asyn(lvfp,0,0)pressure")
+    field(PREC, "3")
+    field(SCAN, ".5 second")
+    field(EGU, "mN/m")
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PRESSURE")
+    field(SDIS, "$(P)DISABLE")
+}
+
+# Write to LabVIEW control "Target  Area" on "C:/LabVIEW Modules/Drivers/Nima Trough/Nima Trough Code/VIs/Nima Trough Interface.vi"
+record(ai, "$(P)AREA:TARGET")
+{
+	field(DESC, "LabVIEW 'Target Pressure'")
```

Comment:
Description for this record (`AREA:TARGET`) is surprising.

Reply:
Updated